### DISCUSSION TITLE
How to create multiple companies ?

### DISCUSSION THREAD

**Discussion by rohitsingal1** [inventree/InvenTree#4865] [Q&A]
I would like to maintain three seperate companies with seperate sets of users, suppliers, buyers and manufacturers. 

One company should not be able to see the data of others.

**Comment by SchrodingersGat** [comment]
We don't currently support multiple tenancy so suggestion would be to maintain three separate InvenTree installations (databases, etc)

**Reply by rohitsingal1** [reply]
How can we maintain multiple instances of inventree on the same server ?

**Reply by matmair** [reply]
You can install it multiple times in docker or - if you have some sysadmin experience with Linux - run the bare metal production setup multiple times with different environment variables for the port and route them with nginx/caddy.

**Reply by rohitsingal1** [reply]
Hi @matmair Docker I always get stuck at getting communication done between docker and host database. Bare metal production is the one for me. 
Will try with different ports. I do believe that apart from the port we need to have seperate databases as well

**Reply by matmair** [reply]
You could also work with different schemas but some security risks still remain if the different companies can not trust each other. Things like plugins will run in a shared context so the cleanest way is docker.
